transform-plugins/widgets/XMLParser-transform.json

```diff
@@ -0,0 +1,74 @@
+{
+  "metadata": {
+    "spec-version": "1.0"
+  },
+  "configuration-groups": [
+    {
+      "label": "XML Parser",
+      "properties": [
+        {
+          "widget-type": "textbox",
+          "label": "Input field to parse as an XML record",
+          "name": "input",
+          "widget-attributes": {
+            "width": "large"
```

Comment:
We don't actually support width(sizing of widget) today. Not sure if this was in the docs.